csrankings-d.csv

```diff
@@ -103,6 +103,7 @@ Dan E. Tamir,Texas State University,http://www.cs.txstate.edu/~dt19,zcdcbaIAAAAJ
 Dan E. Willard,University at Albany - SUNY,https://www.albany.edu/ceas/dan-willard.php,NOSCHOLARPAGE
 Dan Feldman,University of Haifa,http://people.csail.mit.edu/dannyf,67QZN0gAAAAJ
 Dan Feng 0001,HUST,http://faculty.hust.edu.cn/dfeng/en/index.htm,g57OG4QAAAAJ
+Dan Fu,Univ. of California - San Diego,https://danfu09.github.io/,vS2kXmEAAAAJ

```

Comment:
"I'll be joining UCSD as an assistant professor starting January 2026." Cannot be added.

Reply:
OK. Feel free to reject this change. I'll follow up later when they have joined.

Reply:
Please remove from PR.